tendrl/gluster_integration/objects/volume/atoms/stop_profiling/__init__.py

```diff
@@ -27,12 +27,24 @@ def run(self):
                     err
                 )
             )
+        loop_count = 0
         while True:
-            volume = NS.gluster.objects.Volume(
-                vol_id=vol_id
-            ).load()
-            if volume.profiling_enabled == "no":
+            if loop_count >= 24:
+                raise AtomExecutionFailedError(
+                    "Volume profiling not yet marked disabled "
```

Comment:
"Could not disable profiling for volume: %s under cluster: %s, Timed out"

Reply:
ok